indico/modules/events/registration/templates/management/_reglist.html

```diff
@@ -0,0 +1,37 @@
+{% macro render_registrations_list(registrations, visible_cols_regform_items) %}
+    <div class="registrations-table-wrapper">
+        <table class="i-table tablesorter">
+            <thead>
+                <tr class="i-table">
+                    <th class="i-table checkbox-column"></th>
+                    <th class="i-table">{% trans %}Full name{% endtrans %}</th>
+                    {% for item in visible_cols_regform_items %}
+                        <th class="i-table">{{ item.title }}</th>
+                    {% endfor %}
+                </tr>
+            </thead>
+            <tbody>
+                {% for registration in registrations %}
+                    {% set data = registration.data_by_field %}
+                    <tr class="i-table">
+                        <td class="i-table">
+                            <input class="select-row" type="checkbox" name="registrant_id" value="{{ registration.id }}">
+                        </td>
+                        <td class="i-table">
+                            <a href="{{ url_for('.registration_details', registration) }}">
+                                {{- registration.first_name }} {{ registration.last_name -}}
+                            </a>
+                        </td>
+                        {% for item in visible_cols_regform_items %}
+                            <td class="i-table">
+                                {%- if item.id in data %}
```

Comment:
I wouldn't worry about `{{- -}}` unless it's withing `{% trans %}` blocks. I don't think we are trying to build perfectly nice looking html.